unityfiles/Assets/Scripts/Helper/Letter_gen_Finn_pos/SettValuePairs.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+[System.Serializable]
+public class SettValuePairs 
+{
+    public List<SettValuePair> L_1;
+    public List<SettValuePair> L_2;
+    public List<SettValuePair> L_3;
+    public List<SettValuePair> L_4;
+    public List<SettValuePair> L_5;
+    public List<SettValuePair> L_6;
+    public List<SettValuePair> L_7;
+    public List<SettValuePair> L_8;
+    public List<SettValuePair> L_9;
+ 
+    public List<SettValuePair> L_10;
+    public List<SettValuePair> L_11;
+    public List<SettValuePair> L_12;
+    public List<SettValuePair> L_13;
+    public List<SettValuePair> L_14;
+    public List<SettValuePair> L_15;
+    public List<SettValuePair> L_16;
+    public List<SettValuePair> L_17;
+    public List<SettValuePair> L_18;
+    public List<SettValuePair> L_19;
+    public List<SettValuePair> L_20;
+    public List<SettValuePair> L_21;
+    public List<SettValuePair> L_22;
+    public List<SettValuePair> L_23;
+    public List<SettValuePair> L_24;
+    public List<SettValuePair> L_25;
+    public List<SettValuePair> L_26;
+    public List<SettValuePair> L_27;
+    public List<SettValuePair> L_28;
+    public List<SettValuePair> L_29;
+
+    public List<SettValuePair> GetListFromIndex(int index)
+    {
+        switch (index)
+        {
+            case 1:
+                return L_1;
+            case 2:
+                return L_2;
+            case 3:
+                return L_3;
+            case 4:
+                return L_4;
+            case 5:
+                return L_5;
+            case 6:
+                return L_6;
+            case 7:
+                return L_7;
+            case 8:
+                return L_8;
+            case 9:
+                return L_9;
+            case 10:
+                return L_10;
+            case 11:
+                return L_11;
+            case 12:
+                return L_12;
+            case 13:
+                return L_13;
+            case 14:
+                return L_14;
+            case 15:
+                return L_15;
+            case 16:
+                return L_16;
+            case 17:
+                return L_17;
+            case 18:
+                return L_18;
+            case 19:
+                return L_19;
+            case 20:
+                return L_20;
+            case 21:
+                return L_21;
+            case 22:
+                return L_22;
+            case 23:
+                return L_23;
+            default:
+                return L_24;
```

Comment:
Remember we use camelCase...

Reply:
json to class serialization is case sensetive so changing this wold create issues